src/components/Form.jsx

```diff
@@ -35,7 +39,9 @@ function Form() {
         </label>
       </div>
       <div className="btnContainer">
-        <button className="btnBottle" type="button">Ajouter une autre bouteille</button>
+        <div className="btnBottle">
+          { link('/login', 'Se connecter') }
+        </div>
```

Comment:
using the additional parameter in `link`, this may become:

    { link('/login', 'Se connecter', 'btnBottle') }

then no need for the surrounding div ;)